posthog/api/capture.py

```diff
@@ -352,6 +349,25 @@ def get_event(request):
     return cors_response(request, JsonResponse({"status": 1}))
 
 
+def validate_events(events, ingestion_context):
+    for event in events:
+        event_uuid = UUIDT()
+        distinct_id = get_distinct_id(event)
+        payload_uuid = event.get("uuid", None)
+        if payload_uuid:
+            if UUIDT.is_valid_uuid(payload_uuid):
+                event_uuid = UUIDT(uuid_str=payload_uuid)
+            else:
+                statsd.incr("invalid_event_uuid")
+                raise ValueError('Event field "uuid" is not a valid UUID!')
+
+        event = parse_event(event, distinct_id, ingestion_context)
+        if not event:
+            continue
```

Comment:
Why not `raise ValueError` here too?

Reply:
Limiting the scope of the PR/new errors right now - this is the same as previous pattern calling this function.